README.md

```diff
@@ -392,22 +392,41 @@ your-secretRootKey-value:
+| Parameter         | Type     | Description                                                                                                                                               | Valid Values                                                                                                                                            |
+|:-----------------|:---------|:----------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `enabled` | bool | Whether the binding rotation option is enabled.                                                                                                           |                                                                                                                                                         |
+| `rotationFrequency` | string | Indicates the interval between credential rotations. The actual frequency may be longer than the one you specified.  See details under **Notes** below. | "m", "h"                                                                                                                                                |                                                                                                                                                                |
+| `rotatedBindingTTL`   |  string | Specifies for how long to keep the old `ServiceBinding`. The actual TTL may be longer than the one you specified.  See details under **Notes** below.   | "m", "h"                                                                                                                                                |   
 
-**Note:**<br> It isn't possible to enable automatic credentials rotation to an already-rotated `ServiceBinding` (with the `services.cloud.sap.com/stale` label).
+### Example
+In the following example, the `ServiceBinding` resource is configured to rotate credentials every 25 days and to keep the old `ServiceBinding` for 2 days before deleting it.
+```yaml
+apiVersion: services.cloud.sap.com/v1
+kind: ServiceBinding
+metadata:
+  name: sample-binding
+spec:
+  serviceInstanceName: sample-instance
+  credentialsRotationPolicy:
+    enabled: true
+    rotatedBindingTTL: 48h
+    rotationFrequency: 600h
+ ```
+
+**Notes**
+* **The `credentialsRotationPolicy` is evaluated and executed during the [control loop](https://kubernetes.io/docs/concepts/architecture/controller/) which runs on every update or during
+  a full reconciliation process. This means that the actual rotation time may potentially be greater than the specified duration in 'rotationFrequency'.**
```

Comment:
I would move this note into the field descriptions in the properties table above, even twice (for each field).
Try to make it a bit shorter.